Geospatial.md

```diff
@@ -162,3 +197,26 @@ The axis order of the coordinates in WKB and bounding box stored in Parquet
 follows the de facto standard for axis order in WKB and is therefore always
 (x, y) where x is easting or longitude and y is northing or latitude. This
 ordering explicitly overrides the axis order as specified in the CRS.
+
+# Special geospatial values
+
+A special geospatial value refers to an individual axis value (e.g., X, Y, Z,
```

Comment:
Is "axis value" a standard term? Could we use something more clear, like "scalar value" instead?

Reply:
Thanks. Fixed accordingly.